awxkit/awxkit/api/pages/api.py

```diff
@@ -1,6 +1,48 @@
+import itertools
+import logging
+import queue
+
 from awxkit.api.resources import resources
+import awxkit.exceptions as exc
 from . import base
 from . import page
+from .. import utils
+from ..mixins import has_create
+
+log = logging.getLogger(__name__)
+
+
+EXPORTABLE_RESOURCES = [
+    'users',
+    'organizations',
+    'teams',
+    'credential_types',
+    'credentials',
+    'notification_templates',
+    'projects',
+    'inventory',
+    'inventory_sources',
```

Comment:
Trying to export an inventory source I am seeing the following on the output:

```
This endpoint is deprecated: /api/v2/inventory_scripts/1/
Foreign key 'source_script' export failed for object /api/v2/inventory_sources/24/, setting to null
```

That means that the export functionality is partially exporting inventory sources that are created with a custom inventory script. When importing then, it will bring the information but not the script so it won't be able to sync that inventory source after the import process.

All that said, is this an intended behavior? Should we avoid export custom scripts inventory sources entirely? Should we provide a better message saying that the script won't be imported and that it would require some manual work?

Reply:
noting here for posterity: it turns out that the default behavior of the `cli` fixture in tower-qa is to lump together stdout and stderr.  This is not what we want to do for these tests, since as noted it will break json deserialization if there are _any_ log messages.

Reply:
Tests will be updated after confirming that it is fine to not fully support inventory sources with inventory scripts.

The point is that the current functionality will export partial inventory sources when they have an inventory script on its current state. And those won't be able to sync after being imported because it will be missing the inventory script that was used to sync previously.